Documentation/disaster-recovery.md

```diff
@@ -17,35 +17,114 @@ For example, if you have three mons and lose quorum, you will need to remove the
 that it is the only mon in quorum, and then restart the good mon.
 
 ### Stop the operator
+
 First, stop the operator so it will not try to failover the mons while we are modifying the monmap
 ```bash
-kubectl -n rook-ceph delete deployment rook-ceph-operator
+kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas=0
 ```
 
 ### Inject a new monmap
+
 **WARNING: Injecting a monmap must be done very carefully. If run incorrectly, your cluster could be permanently destroyed.**
 
 The Ceph monmap keeps track of the mon quorum. We will update the monmap to only contain the healthy mon.
 In this example, the healthy mon is `rook-ceph-mon-b`, while the unhealthy mons are `rook-ceph-mon-a` and `rook-ceph-mon-c`.
 
-Connect to the pod of a healthy mon and run the following commands.
+Take a backup of the current `rook-ceph-mo-b` Deployment:
+```bash
+kubectl -n rook-ceph get deployment rook-ceph-mon-b -o yaml > rook-ceph-mon-b-deployment.yaml
+```
+
+Open the file and copy the `command` and `args` from the `mon` container (see `containers` list). This is needed for the monmap changes.
+Cleanup the copied `command` and `args` fields to form a pastable command.
+Example:
+
+```yaml
+[...]
+  containers:
+  - args:
+    - --fsid=41a537f2-f282-428e-989f-a9e07be32e47
+    - --keyring=/etc/ceph/keyring-store/keyring
+    - --log-to-stderr=true
+    - --err-to-stderr=true
+    - --mon-cluster-log-to-stderr=true
+    - '--log-stderr-prefix=debug '
+    - --default-log-to-file=false
+    - --default-mon-cluster-log-to-file=false
+    - --mon-host=$(ROOK_CEPH_MON_HOST)
+    - --mon-initial-members=$(ROOK_CEPH_MON_INITIAL_MEMBERS)
+    - --id=a
+    - --setuser=ceph
+    - --setgroup=ceph
+    - --foreground
+    - --public-addr=10.100.13.242
+    - --setuser-match-path=/var/lib/ceph/mon/ceph-a/store.db
+    - --public-bind-addr=$(ROOK_POD_IP)
+    command:
+    - ceph-mon
+[...]
+```
+
+To this (**do not copy this command**):
```

Comment:
What is this for?

Reply:
People should not copy the example command, as they need to "create" there own.